meetings/2020-11-10.md

```diff
@@ -24,60 +24,63 @@
     - Audrie: touch base with HiRISE and CaSSIS teams
       - got in touch with both, hirise+cassis was posted.
     - Victor: touch base with LROC team
-      - LROC, monthly team meeting nov 11th will be posted that day.
+      - LROC, monthly team meeting nov 11th will be posted that day
+      - Collecting input from entire LROC team including Emmerson
     - Jesse: touch base with ASC and twitter
-      - no updates
-  - Release WG (no updates)
-    - Jesse: Get more people to participate in the releases WG
-    - Jesse: Kick off release WG
+      - Set out internal e-mail
+      - Posted on astrogeology twitter
+      - Didn't seem to get any more input
+
+- Release Working Group
+  - Hasn't started working yet
+  - Do we have enough information?
+    - Uncertain if we do
+    - Uncertain if we will be able to get more
+    - There are still people working on writing stories
+      - Kris & LROC are known people still writing
+    - The working group should decide if they have enough
+      - If they do, they should go ahead and move forward prior to reporting
+  - Should stories be from user or developer perspective?
+    - Separate stories for each perspective/persona
+    - We want to support both developers and users
 
 - Collecting "epic" changes
   - examples:
     - OpenCV
-  - nov 10th we don't really know what this means...
-    - lacking context bump to next time
-
-- Discussion if we want to talk about release stories today or next meeting
-  - release working group, if that group is ready...
-  - sounds like next time....
-  - next time...
-  - Kris - has some stories, developer vs user perspective question? - Jay said both I think, but external developer more important for now (we have less of it). other comments for other places....
-  - Victor, also has developer and user perspective. 
+  - lacking context bump to December meeting
 
 - Governance for ISIS
-  - ISIS is moving through TC
-  - Gov model for ISIS, need to define it for the TC application
-  - Who updates the language to update ISIS TC - Jesse
+  - ISIS is moving through TSC
+    - Track progress [On the TSC repo](https://github.com/planetarysoftware/TSC/issues/113)
+      - Decision making model needs to be clarified
+      - Code of Conduct is still a dead link
+      - Contributing depends upon code of conduct
+  - Need to update the application and clarify how the ISIS TC works
 
 ## Discussions for next meeting
 
 - Action Items from last meeting
 
+- Release Working Group
+
+- Collecting Epic Changes
+
 ## Action Items
 
-- Victor 
-  - LROC meeting the 11th, user & dev stories to contribute by next meeting 
-- Jesse 
-  - Release WG
-  - have something started and to talk about at next meeting, report to TC about membership,
+- Victor
+  - LROC meeting the 11th, user & dev stories to contribute by next meeting
+- Kris
```

Comment:
@KrisBecker This wasn't on the original action items but was discussed during the meeting. I can also change this to be stories from the ORex team.

Reply:
@jessemapel Yeah, I mentioned that I would be able to put together a couple of stories from our experience. I have canvased our team for input so that will be forthcoming soon. Thanks...